packages/babel-preset-env/test/fixtures/debug/browserslists-defaults-not-ie/stdout.txt

```diff
@@ -16,21 +16,11 @@ Using targets:
 Using modules transform: auto
 
 Using plugins:
-  transform-unicode-sets-regex { chrome < 112, firefox < 116, ios, opera_mobile < 75, safari, samsung }
+  transform-unicode-sets-regex { chrome < 112, firefox < 116, ios, opera_mobile < 75, safari < tp, samsung }
   transform-class-static-block { ios < 16.4, safari < 16.4 }
-  syntax-private-property-in-object
-  syntax-class-properties
-  syntax-numeric-separator
-  syntax-nullish-coalescing-operator
-  syntax-optional-chaining
-  syntax-json-strings
-  syntax-optional-catch-binding
-  syntax-async-generators
-  syntax-object-rest-spread
```

Comment:
These changes (for Babel 8 tests) should have been included in previous PRs, I don't know why theses tests were passing without changes here.